### ISSUE TITLE
Don't guess at transfers during batch elimination during a datatables upload

### ISSUE THREAD

**Issue by artoonie** [artoonie/rcvis#483]
see thread: https://x.com/nealmcb/status/1726788771465597164?s=20

**Comment by nealmcb** [comment]
Thank you, I strongly agree that the tool should not invent data or assume proportionality. And producing non-integer vote counts raises a whole nother set of issues.

More broadly speaking, one of the points of any improved voting method is to help voters to express their preferences more clearly, and to use that to better understand the electorate and achieve more effective democracy.  Batch elimination strikes a blow against that goal by hiding the actual preferences of voters for each of the candidates eliminated in a batch. Surely one of the reasons that people do batch elimination is to reduce the size and complexity of data presentations. If RCVis was able to take sequential elimination data and allow the viewer to show it as batch elimination, we would hopefully get more jurisdictions producing all the data with its rich information about voter preferences, yet and let viewers choose how much detail to see. That would also allow batch elimination with proper transfer flow visualizations.

**Comment by artoonie** [comment]
Agreed. This only happened during manual data entry into the RCVis upload box (https://mailchi.mp/d690d27b90b0/feature-announcement-manual-data-entry-for-rcv-visualizations). It has been resolved with #484
